samples/cellular/modem_shell/README.rst

```diff
@@ -1182,79 +1160,50 @@ Application FOTA support
 To build the MoSh sample with application FOTA support, use the ``-DOVERLAY_CONFIG=overlay-app_fota.conf`` option.
 For example:
 
-.. tabs::
-
-   .. group-tab:: nRF9161 DK
+.. parsed-literal::
+   :class: highlight
 
-      .. code-block:: console
+   west build -p -b *build_target* -d build -- -DOVERLAY_CONFIG=overlay-app_fota.conf
 
-         west build -p -b nrf9161dk_nrf9161_ns -d build -- -DOVERLAY_CONFIG=overlay-app_fota.conf
-
-   .. group-tab:: nRF9160 DK
-
-      .. code-block:: console
-
-         west build -p -b nrf9160dk_nrf9160_ns -d build -- -DOVERLAY_CONFIG=overlay-app_fota.conf
+|build_target|
 
 nRF91 Series DK with full modem FOTA support
 ============================================
 
 To build the MoSh sample for an nRF91 Series DK with full modem FOTA support, use the devicetree overlay for external flash corresponding to your device and the ``-DOVERLAY_CONFIG=overlay-modem_fota_full.conf`` option.
-For example:
+The following is an example for the nRF9161 DK:
 
-.. tabs::
-
-   .. group-tab:: nRF9161 DK
-
-      .. code-block:: console
-
-         west build -p -b nrf9161dk_nrf9161_ns -d build -- -DOVERLAY_CONFIG=overlay-modem_fota_full.conf -DDTC_OVERLAY_FILE=nrf9161dk_ext_flash.overlay
-
-   .. group-tab:: nRF9160 DK
+.. code-block:: console
 
-      .. code-block:: console
+   west build -p -b nrf9161dk_nrf9161_ns -d build -- -DOVERLAY_CONFIG=overlay-modem_fota_full.conf -DDTC_OVERLAY_FILE=nrf9161dk_ext_flash.overlay
 
-         west build -p -b nrf9160dk_nrf9160_ns -d build -- -DOVERLAY_CONFIG=overlay-modem_fota_full.conf -DDTC_OVERLAY_FILE=nrf9160dk_ext_flash.overlay
+|build_target|
```

Comment:
This example command doesn't have `*build_target*` in it, so the shortcut wording is wrong.